src/docs/product/accounts/user-settings/index.mdx

```diff
@@ -5,78 +5,44 @@ redirect_from:
 
-Click the **Session History** tab to find a historical list of your sessions, including First Seen and Last Seen timestamps.
+<Note>
+<markdown>
+Clicking "Sign out of all devices" will end your sessions with any device logged in to Sentry using your account.
+</markdown>
+</Note>
 
 ## Notifications
 
-Sentry notifications are broadly categorized into alerts and non-alert notifications.
-
-- Alerts notify you about problems in your application and can be sent to many supported [integrations](/product/integrations/#webhook-alerts). You can set alerts on both errors and performance data.
-- Non-alert are all other types of Sentry notifications such as workflow notifications, deploy notification, and quota & usage notifications. These are sent only email.
+Notification emails send when an issue state changes, a release deploys, or there's a spike in your quota. Fine-tune notifications per project by navigating to [Workflow Notifications](https://sentry.io/settings/account/notifications/workflow/). For more details about Notifications, see our [full documentation on Alerts & Notifications](/product/alerts-notifications/).
 
-For more details, see our [full documentation on Alerts & Notifications](/product/alerts-notifications/).
+### Email Routing
 
-Define your preference for **Alerts** and **Workflow Notifications**. Fine-tune the notifications per project using the dropdown selector.
+[Email Routing](https://sentry.io/settings/account/notifications/email/) controls the emails sent for each project. These notifications default to the email address used to set up your Sentry account. Route email notifications to an alternative email address to organize your project notifications. For example, infrastructure-related projects can route to an email dedicated to your team's infrastructure.
 
-**Email Routing** preference controls the emails sent for each project. These notifications default to the email address used to set up your Sentry account. Route email notifications to an alternative email address to organize your project notifications. For example, infrastructure-related projects can route to an email dedicated to your team's infrastructure.
+### Weekly Reports
 
-Sentry generates reports **Weekly Reports** per project and sends these reports once a week on Mondays. Fine-tune your reports per project using the dropdown selector. The following report sample shows details on Events Seen This Week, Events by Issue Type, Events by Project, and activity over time.
+Sentry generates [Weekly Reports per project](https://sentry.io/settings/account/notifications/reports/) and sends these reports once a week on Mondays. The following report sample shows details on Events Seen This Week, Events by Issue Type, Events by Project, and activity over time.
```

Comment:
Suggest we modify the content here to make it slightly more evergreen should our reports change (so we'd only need to swap out the shot)
```suggestion
Sentry generates [Weekly Reports per project](https://sentry.io/settings/account/notifications/reports/) and sends these reports once a week on Mondays. For example:
```